PROTOCOL.authorizations

```diff
@@ -44,7 +44,10 @@ about the matching capabilities.
 
 The reserved keys are evaluated as follows:
 
-* role: evaluated as a normal constraint against the requested role.
+* role: evaluated as a normal constraint against the requested role. The
+  special value @SELF will be automatically expanded to the list of
```

Comment:
Would something like `@PRINCIPALS` maybe be more descriptive? Since the list of principals might be e.g. "root" and not necessarily the user themselves.

Reply:
Fair point, done.